### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by 2e2a**
Makes the dashboard work.

Depends on: https://github.com/liqd/a4-meinberlin/pull/844

TODO:

- [x] Fix Image preview in modal
- [x] Make document and poll editors work
- [x] fix styling (some element alignment looks strange

### CODE REVIEW

File: liqd_product/config/settings/base.py [hunk 1]
```diff
@@ -298,5 +303,10 @@
 A4_MAP_ATTRIBUTION = '&copy; <a href="http://openstreetmap.org/copyright">OpenStreetMap</a> contributors'
 A4_MAP_BOUNDING_BOX = ([[52.3517, 13.8229], [52.6839, 12.9543]])
 
+A4_DASHBOARD = {
+    'PROJECT_DASHBOARD_CLASS': 'meinberlin.apps.dashboard2.ProjectDashboard',
```
Comment (kleingeist) [inline liqd_product/config/settings/base.py]: should point to liqd_product...

Comment (2e2a) [reply liqd_product/config/settings/base.py]: I thought we could use the default core dashboard until needed otherwise.

Comment (kleingeist) [reply liqd_product/config/settings/base.py]: the TypedDashboard is required to create/edit external projects

Comment (2e2a) [reply liqd_product/config/settings/base.py]: I was not sure that external projects are required in the product.

Comment (kleingeist) [reply liqd_product/config/settings/base.py]: okay, thats a good point :) then we'd be able to drop the whole custom dashboard logic. but we should remove external projects from base.py then

**Comment by xi** [comment]
`partner` is not set in the dashboard views.

**Comment by xi** [comment]
@2e2a the webpack issue should be fixed with fe3a73d

The image upload issue should be fixed with https://github.com/liqd/adhocracy4/pull/191

**Pull request merged**